scripts/report-bundle-size/report-as-a-pr-comment.js

```diff
@@ -7,6 +7,8 @@ const LOCAL_BRANCH = process.env.CI_COMMIT_REF_NAME
 const PR_COMMENTER_AUTH_TOKEN = command`authanywhere`.run().split(' ')[2].trim()
 const GITHUB_TOKEN = getGithubAccessToken()
 const ONE_DAY_IN_SECOND = 24 * 60 * 60
+const SIZE_INCREASE_THRESHOLD = 5
+// The value is set to 5% as it's around 10 times the average value for small PRs.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** comments are usually above the line they refer to 🤔 
at least for consistency across the repo let's move it above
```suggestion
// The value is set to 5% as it's around 10 times the average value for small PRs.
const SIZE_INCREASE_THRESHOLD = 5
```